resources/core/charts/environments/templates/4-deployment.yaml

```diff
@@ -22,7 +22,11 @@ spec:
       serviceAccountName: {{ template "name" . }}-service-account
       containers:
         - name: {{ .Chart.Name }}
+        {{ if .Values.global.environments.dir }}
```

Comment:
can we implement it in cleaner way? To build dir variable in a way that this if else statement won't be needed here: e.g. `dir="develop/"` so in chart we would have something like: `image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.environments.dir }}environments:{{ .Values.global.environments.version }}` and we would handle `dir=""` in the same way.

Reply:
Applied the change, please take another look.
Thanks!